src/main/java/org/example/bookstore/service/category/CategoryServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package org.example.bookstore.service.category;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.example.bookstore.dto.categorydto.CategoryDto;
+import org.example.bookstore.dto.categorydto.CategoryRequestDto;
+import org.example.bookstore.exception.CategoryException;
+import org.example.bookstore.exception.EntityNotFoundException;
+import org.example.bookstore.mapper.CategoryMapper;
+import org.example.bookstore.model.Category;
+import org.example.bookstore.repository.category.CategoryRepository;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CategoryServiceImpl implements CategoryService {
+    private final CategoryRepository categoryRepository;
+    private final CategoryMapper categoryMapper;
+
+    @Override
+    public List<CategoryDto> findAll(Pageable pageable) {
+        return categoryRepository
+                .findAll()
+                .stream()
+                .map(categoryMapper::toDto)
+                .toList();
+    }
```

Comment:
Here you also don't use pageable
```suggestion
    @Override
    public List<CategoryDto> findAll(Pageable pageable) {
        return categoryRepository
                .findAll(pageable)
                .stream()
                .map(categoryMapper::toDto)
                .toList();
    }
```